### PR TITLE
Adding approve plugin to ocp-release-operator-sdk

### PR DESCRIPTION
**Pull request by jmrodri**

### CODE REVIEW

File: core-services/prow/02_config/openshift/ocp-release-operator-sdk/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+plugins:
+  openshift/builder:
```
Comment (bparees) [inline core-services/prow/02_config/openshift/ocp-release-operator-sdk/_pluginconfig.yaml:2]: this needs to be your repo name

Comment (joelanford) [inline core-services/prow/02_config/openshift/ocp-release-operator-sdk/_pluginconfig.yaml:2]:
```suggestion
  openshift/ocp-release-operator-sdk:
```

**Review by joelanford — COMMENTED** [review]
/lgtm

**Pull request merged**